lib/centurion/docker_server.rb

```diff
@@ -18,15 +18,15 @@ class Centurion::DockerServer
                  :remove_container, :restart_container
   def_delegators :docker_via_cli, :pull, :tail, :attach, :exec, :exec_it
 
-  def initialize(host, docker_path, tls_params = {})
+  def initialize(host, docker_path, params = {})
```

Comment:
Could this be `cxn_params` or `connection_opts`? `params` is up there with `data` for bad varnames.